URA-695/vcf_making.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash/
```

Shellcheck (0.10.0)

[error] This shebang specifies a directory. Ensure the interpreter is a file.

(SC2246)